db/migrations/20220109175923_user.sql

```diff
@@ -0,0 +1,14 @@
+-- +goose Up
+CREATE TABLE IF NOT EXISTS public."User"
+(
+    "user_ID" integer NOT NULL GENERATED ALWAYS AS IDENTITY ( INCREMENT 1 START 1 MINVALUE 1 MAXVALUE 2147483647 CACHE 1 ),
+    "login" text COLLATE pg_catalog."default" NOT NULL,
+    "password" text COLLATE pg_catalog."default" NOT NULL,
+    "add_halls" boolean NOT NULL,
```

Comment:
What if we need to add another privilege?

Reply:
add another column?

Reply:
And modify the Resource struct in go code?
I need it to work dynamically

Reply:
maybe i could use map? trying to find a best practsice to strore it 🤔

Reply:
table `users`
table `privileges`
table `user_privileges`

```
Select p.*
from privileges p
join user_privileges u_p on u_p.privilege_id = p.id
where up.user_id = <666>
```

Something like that